src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+  --big-search-height: 70px;
+  --small-search-height: 42px;
+}
+
+.search-form {
+  height: var(--big-search-height);
```

Comment:
we can set the height for the input, and the form will simply stretch out

Reply:
✅